### PR TITLE
Account actor

### PR DESCRIPTION
**Pull request by turuslan**
### Description of the Change
Add `AccountActorState` and CBOR encode/decode.
Add `AccountActor::create`, `AccountActor::resolveToKeyAddress`.
Add `StateTree.store`.
Add `setupInitActor` to testutil.

### Benefits
Account actor

### Possible Drawbacks 
None

### CODE REVIEW

File: test/testutil/init_actor.hpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#ifndef CPP_FILECOIN_TEST_TESTUTIL_INIT_ACTOR_HPP
+#define CPP_FILECOIN_TEST_TESTUTIL_INIT_ACTOR_HPP
+
+#include <gtest/gtest.h>
+#include "storage/ipfs/impl/in_memory_datastore.hpp"
+#include "testutil/cbor.hpp"
+#include "vm/actor/init_actor.hpp"
+#include "vm/state/state_tree.hpp"
+
+std::shared_ptr<fc::vm::state::StateTree> setupInitActor(
```
Comment (Harrm) [inline test/testutil/init_actor.hpp]: Please comment.

File: core/vm/actor/account_actor.hpp [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  template <class Stream,
+            typename = std::enable_if_t<
+                std::remove_reference_t<Stream>::is_cbor_decoder_stream>>
+  Stream &operator>>(Stream &&s, AccountActorState &state) {
+    s.list() >> state.address;
+    return s;
+  }
+
+  struct AccountActor {
+    static constexpr VMExitCode CREATE_WRONG_ADDRESS_TYPE{1};
```
[the remark this answers is not in the document]
Comment (turuslan) [reply core/vm/actor/account_actor.hpp]: That's behavior of lotus

https://github.com/filecoin-project/lotus/blob/f5bfaff0033efd0e3639db864b787d631ca57cf5/chain/vm/mkactor.go#L51

https://github.com/filecoin-project/lotus/blob/f5bfaff0033efd0e3639db864b787d631ca57cf5/chain/vm/vm.go#L213

File: core/vm/actor/account_actor.hpp [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    static constexpr VMExitCode RESOLVE_NOT_FOUND{1};
+    static constexpr VMExitCode RESOLVE_NOT_ACCOUNT_ACTOR{1};
+
+    /// Create account actor from BLS or Secp256k1 address
+    static outcome::result<Actor> create(
+        const std::shared_ptr<StateTree> &state_tree, const Address &address);
+    /**
+     * Get BLS address of account actor from ID address
+     * @param state_tree state tree
```
Comment (Harrm) [inline core/vm/actor/account_actor.hpp]: Maybe it better be elaborated on a bit.

File: test/testutil/init_actor.hpp [hunk 4]
```diff
@@ -0,0 +1,30 @@
+#ifndef CPP_FILECOIN_TEST_TESTUTIL_INIT_ACTOR_HPP
+#define CPP_FILECOIN_TEST_TESTUTIL_INIT_ACTOR_HPP
+
+#include <gtest/gtest.h>
+#include "storage/ipfs/impl/in_memory_datastore.hpp"
+#include "testutil/cbor.hpp"
+#include "vm/actor/init_actor.hpp"
+#include "vm/state/state_tree.hpp"
+
+/// Sets up init actor state
```
Comment (Harrm) [inline test/testutil/init_actor.hpp]: And this one, too.

**Pull request merged**